src/blog/2023/01/flowforge-1-3-0-released.md

```diff
@@ -0,0 +1,86 @@
+---
+title: FlowForge 1.3 is now available, share your flows through our new team libraries and much more
+subtitle: Our first release of 2023 with some great new features to try out, happy new year from everyone at FlowForge!
+description: 
```

Comment:
![Screenshot 2023-01-19 at 13 33 11](https://user-images.githubusercontent.com/2529595/213443936-71a3221d-ded1-4fbb-975f-36bbd64f0645.png)

The `null` is shown if this field is empty, can we add something?

Reply:
Fixed in the commit I'm just putting together.